server/views/pages/work.njk

```diff
@@ -28,65 +28,69 @@
+            {% endif %}
+
+            {% if work.createdDate.label %}
+              {% set needsDivider = work.genres.length > 0 or work.subjects.length > 0 or work.lettering.length > 0 or work.descriptionArray.length > 0 %}
+              {% componentV2 'meta-unit', {headingText: 'Date', content: work.createdDate.label, includeDivider: needsDivider} %}
+            {% endif %}
+
+            {% if work.genres.length > 0 %}
+              {% set needsDivider = work.subjects.length > 0 or work.lettering.length > 0 or work.descriptionArray.length > 0 %}
+              {% componentV2 'meta-unit', {headingText: 'Genre', content: work.genres | getLinkObjects('label'), includeDivider: needsDivider} %}
+            {% endif %}
+
+            {% if work.subjects.length > 0 %}
+              {% set needsDivider = work.lettering.length > 0 or work.descriptionArray.length > 0 %}
+              {% componentV2 'meta-unit', {headingText: 'Subject', content: work.subjects | getLinkObjects('label'), includeDivider: needsDivider} %}
+            {% endif %}
+
+            {% if work.lettering.length > 0 %}
+              {% set needsDivider = work.descriptionArray.length > 0 %}
+              {% componentV2 'meta-unit', {headingText: 'Lettering', content: work.lettering, includeDivider: needsDivider} %}
+            {% endif %}
+
+            {% if work.descriptionArray.length > 0 %}
+              <div>
+                <h2 class="{{ {s:'HNM5' , m:'HNM4'} | fontClasses }} {{ {s:0} | spacingClasses({margin: ['top']}) }} {{ {s:1} | spacingClasses({margin: ['bottom']}) }}">
+                  Description
+                </h2>
+                {% for paragraph in work.descriptionArray %}
+                <p class="{{ {s:'HNL5' , m:'HNL4'} | fontClasses }} {{ {s:2} | spacingClasses({margin: ['bottom']}) }}">{{paragraph}}</p>
+                {% endfor %}
+              </div>
+            {% endif %}
+
+            {% if work.encoreLink %}
+              <div class="{{ {s:2} | spacingClasses({margin: ['top']}) }}">
+                {% componentV2 'more-info-link', {name: 'View Wellcome Library catalogue record', url: work.encoreLink} %}
+              </div>
+            {% endif %}
+          </div>
 
-          {% if work.encoreLink %}
-            <div class="{{ {s:2} | spacingClasses({margin: ['top']}) }}">
-              {% componentV2 'more-info-link', {name: 'View Wellcome Library catalogue record', url: work.encoreLink} %}
-            </div>
-          {% endif %}
+          {% set licenseInfo = work.thumbnail.license.licenseType | getLicenseInfo %}
+          {% set usingThisImageData = [
+            {title: 'License infomation', content: licenseInfo.humanReadableText},
+            {title: 'Credit', content: work.attribution}
+          ] %}
 
+          {% componentV2 'work-drawer', {id: 'work-using-image', heading: 'Using this image'}, null, {data: usingThisImageData, topDivider: true} %}
```

Comment:
I wonder if something like "How to use this image" might make more sense as an answer to a question...